lib/authlogic/session/callbacks.rb

```diff
@@ -63,7 +63,10 @@ module Callbacks
       
       def self.included(base) #:nodoc:
         base.send :include, ActiveSupport::Callbacks
-        if ActiveSupport::VERSION::STRING >= '4.1'
+        if ActiveSupport::VERSION::STRING >= '5'
```

Comment:
Please perform all version comparisons using `Gem::Version`. For example:

```ruby
::Gem::Version.new(::ActiveSupport::VERSION::STRING) >= ::Gem::Version.new('5')
```

Thanks.